src/public/stylesheets/style.css

```diff
@@ -37,13 +37,13 @@ a, a:visited, a:hover {
 }
 
 input, select, textarea {
```

Comment:
Notes are underlined but they did not use to be:

<img width="511" alt="image" src="https://github.com/user-attachments/assets/10faef3a-4d4f-4ca2-a9be-907883a4ee3b">